mne/io/fiff/tests/test_raw_fiff.py

```diff
@@ -1056,6 +1056,18 @@ def test_add_channels():
     assert_array_equal(raw_new[:, :][1], raw[:, :][1])
     assert_true(all(ch not in raw_new.ch_names for ch in raw_stim.ch_names))
 
+    # Testing force updates
+    raw_arr_info = create_info(['1', '2'], raw_meg.info['sfreq'], 'eeg')
+    orig_head_t = raw_arr_info['dev_head_t']
+    raw_arr = np.random.randn(2, raw_eeg.n_times)
+    raw_arr = RawArray(raw_arr, raw_arr_info)
+    # This should error because of conflicts in Info
+    assert_raises(ValueError, raw_meg.copy().add_channels, [raw_arr])
+    raw_conf = raw_meg.copy().add_channels([raw_arr],
```

Comment:
ah good point, I just want to make sure that the function won't throw an error. I updated it to throw away the output, will that work?